### ISSUE TITLE
Re: props.children and props.className

### ISSUE THREAD

**Issue by stevematney** [TWExchangeSolutions/eslint-config-exchange-solutions#13]
These properties have implied definitions in pretty much all React components. Should we really be requiring that they're defined in the `propTypes` definitions? That seems excessive.

**Comment by aabenoja** [comment]
I would agree with this. While I like the rationale that the props being used in the component should be somewhat explicit, `children` and `className` are fairly implicit in how they are used. Most cases there will not be variants in how they are used, so constantly going `children: PropTypes.node` and `className: PropTypes.string` is fairly asinine. If it were the case that their usage is out of the norm I will not be against requiring the validation.

**Comment by mtscout6** [comment]
I'm fine with that. We take Pull Requests :)
